### PR TITLE
Feature/gbi 605 allow different address

### PR DESCRIPTION
**Pull request by coolwather**
##
Was added the ability to the user to add a different address.

### CODE REVIEW

File: src/components/swap/SwapBox.vue [hunk 1]
```diff
@@ -208,6 +214,13 @@ export default defineComponent({
     toggleshowMaxTooltip() {
       this.showMaxTooltip = !this.showMaxTooltip;
     },
+    selectAddressType(type) {
```
Comment (aguschirico) [inline src/components/swap/SwapBox.vue:217]: I'm sorry I'm a bit confused, but I don't see where do you use the selected address in the swapData. Am I missing something?

**Review by aguschirico — CHANGES_REQUESTED** [review]

File: src/components/swap/SwapBox.vue (hunk 1 above)
Comment (coolwather) [reply src/components/swap/SwapBox.vue:217]: This method is used to change the type of address, connected -> Metamask or different -> Any other address added by user.

Comment (aguschirico) [reply src/components/swap/SwapBox.vue:217]: and where do you send that in the call to the api?

**Review by aguschirico — APPROVED** [review]
LGTM!

**Pull request merged**